src/LienToken.sol

```diff
@@ -223,7 +223,7 @@ contract LienToken is ERC721, ILienToken, AuthInitializable, AmountDeriver {
     (lienId, newStack) = _createLien(s, params);
```

Comment:
returning `lienId` is not required anymore.

Reply:
true, though changing this at this juncture would require changing all our tests, acknowledged but will not adjust